script/Deploy.sol

```diff
@@ -119,13 +143,20 @@ contract DeployGeneralisedIncentives is BaseMultiChainDeployer {
         // For each bridge, decode their contracts for each chain.
         for (uint256 i = 0; i < availableBridges.length; ++i) {
             string memory bridge = availableBridges[i];
-            // Get the chains this bridge support.
+            // Get the chains this bridge supports.
             string[] memory availableBridgesChains = vm.parseJsonKeys(bridge_config, string.concat(".", bridge));
             for (uint256 j = 0; j < availableBridgesChains.length; ++j) {
                 string memory chain = availableBridgesChains[j];
-                // decode the address
+                // Decode the address
                 address _bridgeContract = vm.parseJsonAddress(bridge_config, string.concat(".", bridge, ".", chain, ".bridge"));
                 bridgeContract[bridge][chain] = _bridgeContract;
+
+                // Check if the bridge is LayerZero
+                if (keccak256(abi.encodePacked(bridge)) == keccak256(abi.encodePacked("LayerZero"))) {
```

Comment:
@ajimeno04 This does not work.
The reason being that bridge is set after load_config is called. I will fix.

I don't understand how you managed to get a correct deployment given the ULN 0 check.